src/UI/index.html

```diff
@@ -29,6 +29,7 @@
     <link rel="apple-touch-icon" sizes="72x72" href="/Content/Images/touch/72.png"/>
     <link rel="apple-touch-icon" sizes="114x114" href="/Content/Images/touch/114.png"/>
     <link rel="apple-touch-icon" sizes="144x144" href="/Content/Images/touch/144.png"/>
+    <link rel="mask-icon" href="/Logo/sonarr_black_and_white.svg" color="#36C6F4">
```

Comment:
`/Logo` isn't accessible by the web server, it needs to be in `/Content/Images`.